src/store/cooperation/mutations.ts

```diff
@@ -0,0 +1,19 @@
+import { MutationTree } from 'vuex';
+import { CooperationInterface, CooperationMutationTypes, Mutations } from '@/store/cooperation/types';
+import { HTTP } from '@/core/api/http-common';
+import { AxiosResponse } from 'axios';
+
+export const mutations: MutationTree<CooperationInterface> & Mutations = {
+  [CooperationMutationTypes.IS_COOPERATION_REGISTERED]: (state: CooperationInterface, payload) => {
```

Comment:
all requests should be in actions because actions are asynchronous 
mutations only for change state(s)

Reply:
Ok. Do mutations need to be wrapped around by actions with the same name?